sdk/communication/azure-communication-networktraversal/azure/communication/networktraversal/_communication_relay_client.py

```diff
@@ -86,6 +86,7 @@ def from_connection_string(
     def get_relay_configuration(
             self,
             user=None, # type: CommunicationUserIdentifier
+            route_type=None, #type: RouteType
```

Comment:
I would keep the type you can see in the generated code: `Optional[Union[str, RouteType]]`

This is an extensible enum, we do not force people to use the enum.

Also, you need a new `param` in the docstring to represent it